Libraries/PermissionsAndroid/PermissionsAndroid.js

```diff
@@ -92,17 +93,38 @@ class PermissionsAndroid {
       READ_EXTERNAL_STORAGE: 'android.permission.READ_EXTERNAL_STORAGE',
       WRITE_EXTERNAL_STORAGE: 'android.permission.WRITE_EXTERNAL_STORAGE',
     };
+
+    this.RESULTS = {
+      GRANTED: 'granted',
+      DENIED: 'denied',
+      NEVER_ASK_AGAIN: 'never_ask_again',
+    };
   }
 
   /**
+   * DEPRECATED - use query
+   *
    * Returns a promise resolving to a boolean value as to whether the specified
    * permissions has been granted
+   *
+   * @deprecated
    */
   checkPermission(permission: string) : Promise<boolean> {
+    console.warn('"PermissionsAndroid.checkPermission" is deprecated. Use "PermissionsAndroid.query" instead');
+    return Permissions.checkPermission(permission);
+  }
+
+  /**
+   * Returns a promise resolving to a boolean value as to whether the specified
+   * permissions has been granted
+   */
+  query(permission: string) : Promise<boolean> {
```

Comment:
I think it's better to change this to `check` since it doesn't actually return the permission status (which is what I thought at first)